packages/extension-ui/src/components/AnimatedMessage.tsx

```diff
@@ -12,10 +12,10 @@ type Props = {
   in: boolean;
   messageType: MessageType;
   text: string;
```

Comment:
I think it would make sense to have to animated content being passed as children.

Reply:
that's true, but I needed children to be a `string`, so decided against it. I use this to delay text changes, because I didn't find a better way to animate it from one text to another. I think this could be improved, but since we're in hurry and it's not part of this PR I'd leave it

Reply:
I don't think there's anything wrong in maing children `string`, but otherwise sure, makes sense 👍